reasoner/resolution/framework/Resolver.java

```diff
@@ -255,8 +248,8 @@ protected CachingResolutionState(Partial<?> fromUpstream, AnswerCache<ANSWER> an
 
     public static class ExplorationManager {
```

Comment:
I've not really reviewed this because we pair programmed much of it (correct me if I'm wrong)

Reply:
Yes that's true